client/app/hosting/database/hosting-database.service.js

```diff
@@ -92,14 +92,33 @@ angular
          * Get dump details
          * @param {string} serviceName
          * @param {string} name
-         * @param {string} dump
+         * @param {string} dumpId
          */
-        getDump (serviceName, name, dump) {
-            return this.OvhHttp.get(`/hosting/web/${serviceName}/database/${name}/dump/${dump}`, {
+        getDump (serviceName, name, dumpId) {
+            return this.OvhHttp.get(`/hosting/web/${serviceName}/database/${name}/dump/${dumpId}`, {
                 rootPath: "apiv6"
+            }).then((dump) => {
+                dump.snapshotDate = this.constructor.getSnapshotDateOfDump(dump);
+                return dump;
             });
         }
 
+        /**
+         * Get snapshot date of database dump
+         * @param {Object} dump
+         */
+        static getSnapshotDateOfDump (dump) {
+            if (!dump.creationDate) { return undefined; }
```

Comment:
ESLint shouldn't allow this, this is not the way we indent code :). Didn't you have warnings ?

Reply:
One-liner are completely fine, since it could be seen a single instruction.

THERE'S NO WARNING, please pull my code and try on your computer before make some bold assumptions. Look at the test phase with our CI, no problem, only success.

This pull request is in accordance with the project ESLint config